ejector/ejection_transactor.go

```diff
@@ -22,3 +31,223 @@ type EjectionTransactor interface {
 	// Complete the ejection proceedings against the operator with the given address.
 	CompleteEjection(ctx context.Context, addressToEject gethcommon.Address) error
 }
+
+var _ EjectionTransactor = &ejectionTransactor{}
+
+// ejectionTransactor is the production implementation of the EjectionTransactor interface.
+type ejectionTransactor struct {
+	// The address of this ejector instance.
+	selfAddress gethcommon.Address
+
+	// Used to execute eth reads
+	caller *contractEigenDAEjectionManager.ContractIEigenDAEjectionManagerCaller
+
+	// Used to execute eth writes
+	transactor *contractEigenDAEjectionManager.ContractIEigenDAEjectionManagerTransactor
+
+	// A function that can sign transactions from selfAddress.
+	signer bind.SignerFn
```

Comment:
Where'd you dig up this type? It looks like it's part of an optimism libarary- is that actually what we should use?